Add how to benchmark migration in "online mode" (#749)
filecoin-project/lotus-docs: content/en/kb/test-migration/index.md

In the above command, replace `[nv-version-to-migrate-to]` with the network version you want to migrate to. For example, if you want to test the migration to network version 21, you would replace `[nv-version-to-migrate-to]` with `21`.

The last step will create the migration jobs and run through them. You should observe the time it starts, and when it finishes, as well as the system load during this migration.

## Benchmark migration in online mode

> Only run this on a node that you can afford to stop syncing the chain, and can be cleaned up later on.

Reviewer: ```suggestion
Run this only on a node on which you can afford to stop syncing the chain. It can be cleaned up later.
```
